python/pyspark/sql/tests/connect/streaming/test_parity_streaming_foreachBatch.py

```diff
@@ -0,0 +1,36 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pyspark.sql.tests.streaming.test_streaming_foreachBatch import StreamingTestsForeachBatchMixin
+from pyspark.testing.connectutils import ReusedConnectTestCase
+
+
+class StreamingForeachBatchParityTests(StreamingTestsForeachBatchMixin, ReusedConnectTestCase):
+    pass
+
+
+if __name__ == "__main__":
+    import unittest
+    from pyspark.sql.tests.connect.streaming.test_parity_streaming_foreachBatch import StreamingForeachBatchParityTests  # noqa: F401
```

Comment:
@HyukjinKwon I have updated the tests, but still this one seems to be running non-connect tests. Do you see anything odd set up with class?

Reply:
I think it's because there is a typo here here:
```
class StreamingTestsForeachBatchMixin(ReusedSQLTestCase):
```
You don't want the mixin class to extend any class. The spark session is very likely set there

Reply:
Thanks. Just realized that minutes ago :). Fixing.